src/vue/components/options.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div data-alpheios-ignore="all">
+    <div class="alpheios-tab-options-switch">
+        <div class="alpheios-tab-options-switch--item" :class="{ 'alpheios-active': currentTab === 1 }" @click="currentTab = 1">U</div>
+        <div class="alpheios-tab-options-switch--item" :class="{ 'alpheios-active': currentTab === 2 }" @click="currentTab = 2">F</div>
+        <div class="alpheios-tab-options-switch--item" :class="{ 'alpheios-active': currentTab === 3 }" @click="currentTab = 3">R</div>
+    </div>

```

Comment:
what do you think about adding a hint just above the U/F/R buttons which says "Select an option set ('U' for user interface options, 'F' for feature options and 'R' for resource options):" ?

Reply:
Sorry, Bridget! I have forgotten about this!
Fixed.

Reply:
ok, but if we're going to do it as tooltips instead of a hint, let's change the wording of the toolips to just "User Interface Options", "Feature Options" and "Resource Options". I am not sure if a hint isn't needed still, but let's see what Monica says when she tests. Thanks!

Reply:
Bridget, may be I didn't understand you properly ...
What is a hint?
Could you show me an example in some other place on a panel or popup?

Reply:
well, this is a good point -- because we don't really use hints elsewhere in the interface (and where we've had them, as in the texts interface, Harry keeps asking me to take them out...).  So let's just stick with the tooltips for now, rewording just to describe each option as I suggested. Thank you!

Reply:
ok - fixed :)